src/compile/scale/range.ts

```diff
@@ -221,7 +226,12 @@ export function defaultRange(
         // Only nominal data uses ordinal scale by default
         return type === 'nominal' ? 'category' : 'ordinal';
       } else if (isContinuousToDiscrete(scaleType)) {
-        return 'ordinal';
+        if (config.range && config.range.ordinal) {
+          return 'ordinal';
+        } else {
+          // for now 4 is the default value for range cardinality. we might change it later
+          return {scheme: 'blues', count: 4};
```

Comment:
Should it be 4 for all of the quantile, quantize, threshold?

Reply:
(It makes sense for quantile, but I doubt if it is for quantize.)

Reply:
This 4 should be in the config too.  Maybe `config.scale.quantileCount`

Reply:
FWIW, the [default `ordinal` scheme in Vega](https://github.com/vega/vega-parser/blob/master/src/config.js#L201) uses `extent: [0.2, 1]` too